### PR TITLE
feat: add copy to clipboard in example block

### LINKED ISSUE #144: Add copy button to sandbox example code block
Add clipboard button here:
image

Use Icon info dialog as a reference:
image

### PR DESCRIPTION
**Pull request by NikitaCG**
close #144

### CODE REVIEW

File: src/components/MDXRenderer/ExampleBlock/ExampleBlock.tsx [hunk 1]
```diff
@@ -54,9 +56,24 @@ export const ExampleBlock: React.FC<ExampleBlockProps> = ({code, background, chi
             </div>
             {codePrepared ? (
                 <div className={b('code', {open: isOpen})}>
-                    <pre>
-                        <code className="language-tsx">{codePrepared}</code>
-                    </pre>
+                    <ClipboardArea
```
Comment (imsitnikov) [inline src/components/MDXRenderer/ExampleBlock/ExampleBlock.tsx:59]: Could you add `margin-right` to the code block for clipboard icon?

Now:
image

After adding margin:
image

**Review by imsitnikov — APPROVED** [review]

File: src/components/MDXRenderer/ExampleBlock/ExampleBlock.tsx (hunk 1 above)
Comment (NikitaCG) [reply src/components/MDXRenderer/ExampleBlock/ExampleBlock.tsx:59]: Yes. In this case, is it necessary to leave a background image?

Comment (imsitnikov) [reply src/components/MDXRenderer/ExampleBlock/ExampleBlock.tsx:59]: Do you mean background color? Yes, it would be better to leave it.

Comment (NikitaCG) [reply src/components/MDXRenderer/ExampleBlock/ExampleBlock.tsx:59]: done

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**